jscomp/bsb/templates/basic-reason/bsconfig.json

```diff
@@ -14,5 +14,6 @@
   "bs-dependencies": [
       // add your dependencies here. You'd usually install them normally through `npm install my-dependency`. If my-dependency has a bsconfig.json too, then everything will work seamlessly.
   ],
-  "namespace": true
+  "namespace": true,
+  "refmt": 3
```

Comment:
would you do a rebase against master, seems this is already done in the master

Reply:
fixed :)  looks like `react-examples` didn't updated yet.